java/src/main/java/ai/rapids/cudf/Schema.java

```diff
@@ -120,7 +120,7 @@ private void flattenIfNeeded() {
 
   private int flattenedLength(int startingLength) {
     if (childSchemas != null) {
-      for (Schema child: childSchemas) {
+      for (Schema child : childSchemas) {
```

Comment:
The IDE formats the entire file, which causes some unrelated changes.

Reply:
Can you please remove the changes or update the IDE then?

Reply:
I prefer to keep the changes as they just reformat the (incorrect) code style which was not formatted properly.

Reply:
It was formatted properly. The space in the for loop is not part of the formatting guidelines. It is something you IDE decided it should add.